ql/src/test/queries/clientpositive/create_table.q

```diff
@@ -0,0 +1,5 @@
+set hive.support.concurrency=true;
+set hive.txn.manager=org.apache.hadoop.hive.ql.lockmgr.DbTxnManager;
+set hive.create.as.external.legacy=true;
+create managed table test as select 1;
+show create table test;
```

Comment:
could you please add 
1. "describe formatted test" to this test case also?
2. your expectation in a comment (before show create table)?

Reply:
Updated the test files as you request and removed the TestMiniTezCliDriver test.